gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js

```diff
@@ -1049,6 +1049,20 @@ GbGradeTable.renderTable = function (elementId, tableData) {
     dropdownMenu.classList.add("gb-dropdown-menu");
 
     dropdownMenu.dataset.cell = link.closest(".tabulator-cell, .tabulator-col");
+
+    document.body.appendChild(dropdownMenu);
+    dropdownMenu.style.position = 'absolute';
+
+    const rect = link.getBoundingClientRect();
+    dropdownMenu.style.top = `${rect.bottom + window.scrollY}px`;
+    dropdownMenu.style.left = `${rect.left + window.scrollX}px`;
+
+    link.addEventListener('hidden.bs.dropdown', function () {
```

Comment:
can you add a comment to explain what's going on here? is this working around a bootstrap bug?

Reply:
Frozen columns and cells have a higher z-index so whenever dropdown menu box exceeds the its parent  element stacking context. its gets hidden by other cell/column. this code moves the dropdown to the body instead